docs/about/contributing.mdx

```diff
@@ -78,10 +80,49 @@ If direct development contribution is not feasible for you, financial support is
 
 We're deeply thankful for the backing from the Open Collective, Open Source Collective, DigitalOcean, and Vercel, which has been instrumental in our project's sustainability and growth:
 
-- **Open Collective & Open Source Collective**: Our collaboration with these organizations offers a robust fiscal management framework and banking solutions, supporting our project alongside over three thousand other open source initiatives. Open Collective's transparent framework lets everyone see our finances, including income, expenditures, and contributions on [Meshtastic's Open Collective page](https://opencollective.com/meshtastic). This transparency and support affirm that we're in excellent company.
-
-- **DigitalOcean**: Champions of open source, DigitalOcean supports Meshtastic through credits, which significantly aid our development, infrastructure, and testing efforts. This contribution is part of their commitment to fostering innovation and collaboration within the open source ecosystem.
-
-![This project is supported by DigitalOcean](https://opensource.nyc3.cdn.digitaloceanspaces.com/attribution/assets/PoweredByDO/DO_Powered_by_Badge_blue.svg "This project is supported by DigitalOcean")
+  <div className="flex flex-wrap items-start">
+    <div className="w-full sm:w-3/12 p-3">
```

Comment:
`w-full` gives 100% width

`sm:w-3/12` means 25% width at screen sizes down to small (768px) in our case - so tablet and up will get two columns while mobile gets 1 stacked

`p-3` is a padding class